cli/tools/fmt.rs

```diff
@@ -687,6 +704,41 @@ fn get_resolved_json_config(
   builder.build()
 }
 
+fn get_resolved_yaml_config(
+  options: &FmtOptionsConfig,
+) -> pretty_yaml::config::FormatOptions {
+  use pretty_yaml::config::*;
+
+  let layout_options = LayoutOptions {
+    print_width: options.line_width.unwrap_or(80) as usize,
+    use_tabs: options.use_tabs.unwrap_or_default(),
+    indent_width: options.indent_width.unwrap_or(2) as usize,
+    line_break: LineBreak::Lf,
```

Comment:
```suggestion
    ..Default::default()
```

Reply:
See https://github.com/denoland/deno/pull/24717#discussion_r1691411918

Reply:
I had overlooked this 🙇